lms/templates/main_django.html

```diff
@@ -10,8 +10,6 @@
 
   {% stylesheet 'style-vendor' %}
   {% stylesheet 'style-main' %}
-  {% stylesheet 'style-course-vendor' %}
-  {% stylesheet 'style-course' %}
```

Comment:
@rlucioni Its not being loaded in ```main.html``` mako version of this template and its effecting on logistration design.